Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -212,6 +212,80 @@ end Stabilizers
 
 end MulAction
 
+section FixedPoints
+
+variable (M : Type u) (α : Type v) [Monoid M]
+
+section Monoid
+
+variable [Monoid α] [MulDistribMulAction M α]
+
+/-- The submonoid of elements fixed under the whole action. -/
+def FixedPoints.submonoid : Submonoid α where
```

Comment:
I feel that `Submonoid.fixedPoints` would be a better name because Lean 4 allows dot notation for them, but we'd need to change the existing FixedPoints.subfield etc. And since you're using the notations `^*` `^+`, there's not much motivation to have good notation. Let me send this to a maintainer to decide whether we want to go that route.

Reply:
I think we can punt on this for now.